lib/utils.ts

```diff
@@ -59,15 +62,27 @@ export function expandTabs(line: string): string {
     });
 }
 
+function getRegexForTempdir(): RegExp {
+    const tmp = os.tmpdir();
+    return new RegExp(tmp.replaceAll('/', '\\/') + '\\/' + ce_temp_prefix + '[\\w\\d-.]*\\/');
+}
+
+/**
+ * Removes the root dir from the given filepath, so that it will match to the user's filenames used
```

Comment:
I'm not quite following: what's the "user's filename" in this context?

Reply:
example.cpp, somesubdir/myclass.cpp, etc

Reply:
got it!